README.md

```diff
@@ -77,6 +77,16 @@ If you wish to execute a backup job once an hour, add the following line to your
 
     0 * * * * cd $HOME/SimplenoteBackup/simplenote-backup && make TOKEN=YOU_TOKEN_HERE > cron.log 2>&1
 
+## Build and run without Python using Docker
+
+    # Build a local Docker image straight from sources on Github.
+    docker build --pull -t simplenote-backup github.com/hiroshi/simplenote-backup
```

Comment:
I didn't aware of that docker build can pull Dockerfile and contexts from git repos, but it worked.
`docker build --pull -t simplenote-backup github.com/pataquets/simplenote-backup`

It is good to know new things. Thanks.